.changeset/lucky-keys-behave.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-jenkins-backend': patch
+'@backstage/plugin-jenkins-common': patch
+---
+
+Expose permissions through the metadata endpoint.
```

Comment:
I think it's better to split the two changesets, keeping this as `jenkins-backend` changeset and adding another one for `jenkins-common` with something like "Export aggregated list of all jenkins permissions"

Reply:
OK, another changeset but still on the same PR? (sorry, not super familiar with the semantics of changesets, I assumed I needed one changeset for the entire PR)